org.mume.MMapper.json

```diff
@@ -0,0 +1,167 @@
+{
+  "app-id": "org.mume.MMapper",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "6.8",
+  "sdk": "org.kde.Sdk",
+  "finish-args": [
+    "--share=network",
+    "--socket=wayland",
+    "--socket=fallback-x11",
+    "--share=ipc",
+    "--device=dri",
+    "--filesystem=xdg-documents:rw"
+  ],
+  "command": "mmapper",
+  "cleanup": [
+    "/app/images",
+    "/app/include",
+    "/app/arda.xml",
```

Comment:
these values are invalid for cleanup....

Reply:
I might be misinterpreting the documentation wrong, but I believe I did it correctly now. Please re-open if I'm still off.